topics/transcriptomics/tutorials/rna-seq-reads-to-counts/workflows/qc_report.ga

```diff
@@ -0,0 +1,442 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
```

Comment:
and to fix the failing travis test, please 
1. fill in the tutorial name in this annotation field
2. add a tags attribute: `"tags": ["transcriptomics"]`